src/supertux/tile_set.cpp

```diff
@@ -55,6 +56,18 @@ TileSet::TileSet() :
   m_tiles[0] = std::make_unique<Tile>();
 }
 
+void
+TileSet::reload()
+{
+  m_autotilesets.clear();
+  m_thunderstorm_tiles.clear();
+  m_tiles.resize(1);
```

Comment:
Why `1` in particular?

Reply:
To preserve the initial tile with an ID of 0. I can add a comment for this.